render_config.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 VAULT_TOKEN=${1:-$(cat "$HOME"/.vault-token)}
-WM_APP_SERVICE_ACCOUNT_VAULT_PATH=secret/dsde/terra/kernel/dev/dev/workspace/app-sa
```

Comment:
This is the service account that gets used for running connected tests. I think you'll also need to update the `google.folder-id` property in the connected test configuration to point to the wsmtest projects folder.

Reply:
Was I supposed to use a new folder ID for that? I copied it from somewhere I think.

Reply:
Terraform generates one, and it should be visible in the atlantis output or from the gcp console. I just double checked, it looks like the wrong one is in helmfile (sorry about that, I completely missed it in review).

Connected tests have a [hardcoded value](https://github.com/DataBiosphere/terra-workspace-manager/blob/dev/src/test/resources/application-connected-test.yml#L3) instead of reading from helm that you'll need to update.

Reply:
Where do I need to update that?

Reply:
https://github.com/DataBiosphere/terra-workspace-manager/blob/dev/src/test/resources/application-connected-test.yml#L3, you should point it to the wsmtest folder ID instead (if that's where we want connected tests to create projects)